packages/amazon-ssm-agent/amazon-ssm-agent.spec

```diff
@@ -8,7 +8,7 @@
 %global goimport %{goproject}/%{gorepo}
 
 Name: %{_cross_os}amazon-ssm-agent
-Version: 3.2.1478.0
```

Comment:
Let's use the version that the ECS agent uses today:

https://github.com/aws/amazon-ecs-agent/blob/master/CHANGELOG.md#1780

Reply:
ECS-agent 1.77.0 uses very old version `3.1.804.0` that is not in sync with SSM agent version(3.2.1478.0) in latest Bottlerocket release. In ECS-agent [1.78.0](https://github.com/aws/amazon-ecs-agent/blob/master/CHANGELOG.md#1780) the SSM agent version will be `3.2.1630.0` but this  ECS-agent version has not been [released](https://github.com/aws/amazon-ecs-agent/releases) yet. So in this update we will keep SSM agent to `3.2.1630.0` version that will bring both of them in sync once we will update ECS agent to 1.78.0.